sale_check_preparation_time/models/sale_order.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Akretion France (http://www.akretion.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import fields, models, api
+import pytz
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    timely_delivery = fields.Boolean(
```

Comment:
When I read it, I assumed timely_delivery means this order should respect a time constraint.
It seems to be the opposit. I suggest to rename it to somethink like "delivered_in_time"